src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,99 @@
-export const Pagination = () => {};
+import { getNumbers } from '../../utils';
+
+type Props = {
+  total: number,
+  perPage: number,
+  currentPage: number,
+  onPageChange: (tabId: number) => void;
+};
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const lastPage = Math.ceil(total / perPage);
+  const pagesArray = getNumbers(1, lastPage);
+  const itemsArray = getNumbers(1, total);
+
+  const itemSeparationForPages = itemsArray.filter(item => {
+    if (currentPage === 1) {
+      return item <= perPage;
+    }
+
+    return item <= perPage * currentPage
+    && item > perPage * (currentPage - 1);
+  });
+
+  return (
+    <>
+      <ul className="pagination">
+        <li
+          className={currentPage === 1 ? 'page-item disabled' : 'page-item'}
+        >
+          <a
+            data-cy="prevLink"
+            className="page-link"
+            href="#prev"
+            aria-disabled={currentPage === 1}
+            onClick={() => {
+              if (currentPage !== 1) {
+                onPageChange(currentPage - 1);
+              }
+            }}
```

Comment:
It's better to move the onClick logic to a separate function. It follows the best practice of separating concerns and keeping your JSX code clean and focused on rendering components rather than defining complex logic directly within it.